docs/maps/vector-style.asciidoc

```diff
@@ -1,6 +1,8 @@
 [role="xpack"]
 [[vector-style]]
-=== Vector style
+=== Vector styling
```

Comment:
### Suggestion

- Add "Style properties" and "Style types" headers and intro sentences.

<img width="807" alt="Screen Shot 2019-04-15 at 12 36 40 PM" src="https://user-images.githubusercontent.com/33642766/56162456-e2617080-5f80-11e9-9df9-71d2e2a9bbb5.png">